_data/snippets.yml

```diff
@@ -614,27 +614,17 @@ translation-target:
   es: Esta lección ha sido traducida al
   fr: Cette leçon a été traduite en
   pt:
-language-name:
-  en:
-    en: English
-    es: inglés
-    fr: anglais
-    pt:
+available-lesson:
+  en: Available in
+  es: Disponible en
+  fr: Disponible en
   pt:
-    en:
-    es:
-    fr:
-    pt:
-  es:
-    en: Spanish
-    es: español
-    fr: espagnol
-    pt:
-  fr:
-    en: French
-    es: francés
-    fr: français
-    pt:
+language-name:
+    en: EN
+    es: ES
+    fr: FR
+    pt: PT
+    
 translation-source:
   en: This lesson was originally published in
```

Comment:
In order to translate "original" you'll need to use one of the snippets to do that, also.

Reply:
I ve made the suggested changes but I haven't finish with it as travis doesnt look happy -- I will work on this alongside the other ticket tomorrow and hopefully will finish it.cheers!

Reply:
@amsichani - I think it's the tabbing in the snippet you added for original in the snippets file. The translations for "original" need to be tabbed over to preserve the hierarchy.